logviewer/conf/conf.go

```diff
@@ -29,6 +30,12 @@ func init() {
 		ContainersHome = "/"
 	}
 
+	kubeMode := os.Getenv("KUBELET_MODE")
+	if kubeMode != "" {
+		log.Println("KUBELET_MODE set")
```

Comment:
prob don't need to print this

Reply:
Was emulating the file, but happy to remove.

Reply:
Yeah, this was the precedent here, but it's a little weird.  It might be more valuable to consolidate all of the `log.Println()`s in this file into one line that prints out their current values.

Reply:
Made a change to use a single logrus field powered log line.  On startup you'll see a line like this:
```
Jan 16 01:32:36 hostname titus-logviewer[3754328]: time="2021-01-16T01:32:36Z" level=info msg="Config loaded" ContainerID= ContainersHome=/var/lib/titus-container-logs KubeletMode=true ProxyMode=true RunningInContainer=false
```